### PR TITLE
LP-2273 Fixing linting issues in custom settings

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2273](https://philanthropyu.atlassian.net/browse/LP-2273)

### Notes

- [x] All linting issue fixed, now rating is 10/10
- [x] All unit test for the app are passing
- [x] iSort ran on while app
- [x] All linting issue posted on the story as a comment

Following issue are fixed
```
************* Module edx-platform.common.djangoapps.custom_settings
common/djangoapps/custom_settings/__init__.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.models
common/djangoapps/custom_settings/models.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/models.py:29: [W0221(arguments-differ), CustomSettings.save] Parameters differ from overridden 'save' method
************* Module edx-platform.common.djangoapps.custom_settings.apps
common/djangoapps/custom_settings/apps.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/apps.py:11: [W0612(unused-variable), CustomSettingsConfig.ready] Unused variable 'custom_settings.signals.handlers'
************* Module edx-platform.common.djangoapps.custom_settings.admin
common/djangoapps/custom_settings/admin.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.urls
common/djangoapps/custom_settings/urls.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.helpers
common/djangoapps/custom_settings/helpers.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/helpers.py:2: [C0411(wrong-import-order), ] standard import "from datetime import datetime" should be placed before "import pytz"
************* Module edx-platform.common.djangoapps.custom_settings.views
common/djangoapps/custom_settings/views.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.signals.handlers
common/djangoapps/custom_settings/signals/handlers.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/signals/handlers.py:14: [W0613(unused-argument), initialize_course_settings] Unused argument 'sender'
common/djangoapps/custom_settings/signals/handlers.py:14: [W0613(unused-argument), initialize_course_settings] Unused argument 'kwargs'
************* Module edx-platform.common.djangoapps.custom_settings.tests.test_handlers
common/djangoapps/custom_settings/tests/test_handlers.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.tests.factories
common/djangoapps/custom_settings/tests/factories.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/tests/factories.py:11: [C0111(missing-docstring), CustomSettingsFactory] Missing class docstring
************* Module edx-platform.common.djangoapps.custom_settings.tests.test_views
common/djangoapps/custom_settings/tests/test_views.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/tests/test_views.py:18: [C0111(missing-docstring), CustomSettingsTestCase] Missing class docstring
************* Module edx-platform.common.djangoapps.custom_settings.tests.test_models
common/djangoapps/custom_settings/tests/test_models.py:1: [C0111(missing-docstring), ] Missing module docstring
************* Module edx-platform.common.djangoapps.custom_settings.tests.test_helpers
common/djangoapps/custom_settings/tests/test_helpers.py:1: [C0111(missing-docstring), ] Missing module docstring
common/djangoapps/custom_settings/tests/test_helpers.py:41: [W0621(redefined-outer-name), test_validate_course_open_date_exceptions] Redefining name 'course_open_date_settings_fixture' from outer scope (line 24)
common/djangoapps/custom_settings/tests/test_helpers.py:55: [W0621(redefined-outer-name), test_validate_course_open_date] Redefining name 'course_open_date_settings_fixture' from outer scope (line 24)

-----------------------------------
Your code has been rated at 9.05/10
```

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: common/djangoapps/custom_settings/__init__.py [hunk 1]
```diff
@@ -1 +1,5 @@
+"""
+Custom settings is an app, and part of course settings in studio, which control specific course functionality. Course
```
Comment (mabdullahz) [inline common/djangoapps/custom_settings/__init__.py:2]: Please consider rephrasing this to `The custom settings app is a part of course settings in studio, which control**s** ...`

**Review by mabdullahz — CHANGES_REQUESTED** [review]

**Review by mabdullahz — APPROVED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: common/djangoapps/custom_settings/helpers.py [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
     """
-    :param settings:
-    :return course open date:
+    Get course open date from settings, if exists, in MM/DD/YYY format else return empty string
+
+    Args:
+        settings (CustomSettings): Custom settings model object
+
+    Returns:
+        string: Date in MM/DD/YYY format or empty string
```
Comment (aliassad) [inline common/djangoapps/custom_settings/helpers.py:21]: Can we change this format detail to  %m/%d/%Y ? it looks confusing to me.

Comment (aliassad) [reply common/djangoapps/custom_settings/helpers.py:21]: you're right %m/%d/%Y looks developer specific, Please change it to "MM/DD/YYYY" as we discussed. Thank you :)

**Review by aliassad — APPROVED** [review]
LGTM 👍

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**